src/current/_includes/releases/v23.2/v23.2.0-beta.3.md

```diff
@@ -0,0 +1,43 @@
+
+<h3 id="v23-2-0-beta-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that could result in an incorrect `too few columns` error for queries that use `ANY <array>` syntax with a subquery. [#115592][#115592]
+- Fixed a bug that could cause `too few columns`/`too many columns` errors for queries that used `IN` or `NOT IN` with a non-trivial right operand, such as a subquery (rather than a constant tuple). [#115592][#115592]
+- Fixed a bug where [`CREATE INDEX`](../v23.2/create-index.html) with expressions could fail on materialized [views](../v23.2/views.html) when the declarative schema changer was used. [#115522][#115522]
+- Fixed a bug that could cause PL/pgSQL routines with `SELECT INTO` syntax to return early. This bug existed only in pre-release versions v23.2.0-beta.1 and v23.2.0-beta.2. [#115676][#115676]
+- Fixed a bug that could cause side effects to happen out of order for PL/pgSQL routines in rare cases. This bug existed only in v23.2 alpha and beta pre-release versions. [#115840][#115840]
+- Previously, in rare cases, CockroachDB could incorrectly evaluate queries with lookup [joins](../v23.2/joins.html) where `equality cols are key` when performing lookups on multiple ranges. This could either manifest as a stuck query or result in incorrect output. The bug was introduced in v22.2 and is now fixed. [#115580][#115580]
+- Fixed a durability bug in Raft log storage that was caused by incorrect syncing of filesystem metadata. It was possible to lose writes of a particular kind (`AddSSTable`) used by (e.g.) `RESTORE`. This loss was possible only under power-off or OS crash conditions. As a result, CockroachDB could enter a crash loop on restart. In the worst case of a coordinated power-off/crash across multiple nodes, this could lead to an unrecoverable loss of quorum. [#115841][#115841]
+- Fixed a bug where large jobs running with execution locality could result in the [gateway node](../v23.2/architecture/sql-layer.html#gateway-node) being assigned most of the work causing performance degradation and cluster instability. [#115876][#115876]
+- Fixed a bug that prevented naming UDT parameters when [dropping a user-defined function](../v23.2/drop-function.html) (or procedure). This bug has existed since v23.1. [#115905][#115905]
+- Locking tables (e.g., with [SELECT ... FOR UPDATE](../v23.2/select-for-update.html)) on the null-extended side of outer joins (e.g., the right side of a `LEFT JOIN`) is now disallowed and returns an error. This improves compatibility with PostgreSQL and prevents ambiguity in locking semantics. This bug has existed since locking with `FOR UPDATE` was introduced. [#115879][#115879]
```

Comment:
```suggestion
- Locking tables (e.g., with [`SELECT ... FOR UPDATE`](../v23.2/select-for-update.html)) on the null-extended side of outer joins (e.g., the right side of a `LEFT JOIN`) is now disallowed and returns an error. This improves compatibility with PostgreSQL and prevents ambiguity in locking semantics. This bug has existed since locking with `FOR UPDATE` was introduced. [#115879][#115879]
```

Reply:
The PR link is not rendering on this one either.